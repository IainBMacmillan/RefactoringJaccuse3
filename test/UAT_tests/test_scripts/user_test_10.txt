Enter
Z
1
Enter
T
Enter
Q