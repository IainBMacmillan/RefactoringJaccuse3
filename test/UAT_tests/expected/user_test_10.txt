J'ACCUSE! (a mystery game)")
    By Al Sweigart al@inventwithpython.com 
    Inspired by Homestar Runner's "Where's an Egg?" game 

    You are the world-famous detective, Mathilde Camus. 
    ZOPHIE THE CAT has gone missing, and you must sift through the clues. 
    Suspects either always tell lies, or always tell the truth. Ask them 
    about other people, places, and items to see if the details they give are 
    truthful and consistent with your observations. Then you will know if 
    their clue about ZOPHIE THE CAT is true or not. Will you find ZOPHIE THE 
    CAT in time and accuse the guilty party? 
    
Press Enter to begin...
 Time left: belongs to testing
 You are in your TAXI. Where do you want to go?
(Q)UIT GAME
> 
 Time left: belongs to testing
 You are at the ZOO.
 SENATOR SCHMEAR with the RAINBOW FLAG is here.

 (J) "J'ACCUSE!" (3 accusations left)
 (Z) Ask if they know where ZOPHIE THE CAT is.
 (T) Go back to the TAXI.
 (1) Ask about SENATOR SCHMEAR
 (2) Ask about RAINBOW FLAG
>  They give you this clue: "No comment."
Press Enter to continue...
 Time left: belongs to testing
 You are at the ZOO.
 SENATOR SCHMEAR with the RAINBOW FLAG is here.

 (J) "J'ACCUSE!" (3 accusations left)
 (Z) Ask if they know where ZOPHIE THE CAT is.
 (T) Go back to the TAXI.
 (1) Ask about SENATOR SCHMEAR
 (2) Ask about RAINBOW FLAG
> Press Enter to continue...
 Time left: belongs to testing
 You are in your TAXI. Where do you want to go?
(Z)OO                  (senator schmear, rainbow flag)
(Q)UIT GAME
> Thanks for playing!
